.editorconfig

```diff
@@ -0,0 +1,12 @@
+root = true
+
+[*]
+indent_size = 4
+indent_style = space
```

Comment:
Will this conflict with `.clang-format`?

Reply:
No because the `TabWidth=4` and `UseTab=Never` in `.clang-format`. Also EditorConfig only affects editor settings and configuration - it does not automatically reformat files. For example, in `ui/xournalpp.css`, it will not change the tabs to spaces - only if you manually de-tab and re-tab those individual lines.